Description: Use "assert Equation" to investigate big-endian mathicgb issues
Author: Doug Torrance <dtorrance@piedmont.edu>
Bug: https://github.com/Macaulay2/M2/issues/2162
Last-Update: 2021-07-02

--- a/M2/Macaulay2/packages/CharacteristicClasses.m2
+++ b/M2/Macaulay2/packages/CharacteristicClasses.m2
@@ -2535,7 +2535,7 @@
     m#{0,1}=csmH#{0,1};
     V=ClassInChowRing(A,I_0);
     seg=Segre(A,ideal(I_0));
-    assert(csmD==csmH#"CSM");
+    assert Equation(csmD, csmH#"CSM");
     assert(csmH#"CSM"==CSM(A,I,m));
     assert(Euler(csmH#"CSM")==7);
     assert(csmH#{0}==Chern(A,ideal(I_0)));
